packages-web/datagrid/src/components/Table.tsx

```diff
@@ -177,19 +184,18 @@ export function Table<T>(props: TableProps<T>): ReactElement {
                 canPreviousPage={canPreviousPage}
                 gotoPage={gotoPage}
                 nextPage={nextPage}
-                page={pageIndex}
+                numberOfItems={props.data.length}
                 numberOfPages={pageOptions.length}
+                page={pageIndex}
+                pageSize={props.pageSize}
                 previousPage={previousPage}
             />
         )
     ) : null;
 
     return (
         <div className={props.className} style={props.styles}>
-            <div className="thead">
-                {props.headerWidgets}
-                {props.columnsHidable && <ColumnSelector allColumns={allColumns} />}
-            </div>
+            <div className="thead">{props.headerWidgets}</div>
```

Comment:
Do we always wanna render thead regardless of undefined headerWidgets ?

Reply:
It was in the wrong element, just moved to the correct thead